support/Machines/Perlmutter.yaml

```diff
@@ -0,0 +1,13 @@
+# Distributed under the MIT License.
+# See LICENSE.txt for details.
+
+Machine:
+  Name: Perlmutter
+  Description: |
+    DOE supercomputer at Lawrence Berkely National Laboratory
+    More information:
+    https://docs.nersc.gov/systems/perlmutter/
+  DefaultTasksPerNode: 8
+  DefaultProcsPerTasks: 32
+  DefaultQueue: "regular"
+  DefaultTimeLimit: "1-00:00:00"
```

Comment:
Not sure, but [docs](https://docs.nersc.gov/jobs/best-practices/#time-limits) say shorter jobs are better on this machine. But that could be more for GPU jobs. @ermost any insight for this?

Reply:
There are some quirks about 2hr jobs being charged less. but by and large Perlmutter behaves like a normal cluster (unlike other DOE machines).